benchmarks/time/src/main/scala/strawman/collection/immutable/HashSetBenchmark.scala

```diff
@@ -21,48 +21,121 @@ class HashSetBenchmark {
 
   var xs: HashSet[Long] = _
   var xss: scala.Array[HashSet[Long]] = _
+  var zipped: HashSet[(Long, Long)] = _
   var randomIndices: scala.Array[Int] = _
+  var randomIndices2: scala.Array[Int] = _
+  var randomXss: scala.Array[HashSet[Long]] = _
 
   @Setup(Level.Trial)
   def initData(): Unit = {
     def freshCollection() = HashSet((1 to size).map(_.toLong): _*)
     xs = freshCollection()
-//    xss = scala.Array.fill(1000)(freshCollection())
-//    if (size > 0) {
-//      randomIndices = scala.Array.fill(1000)(scala.util.Random.nextInt(size))
-//    }
+    xss = scala.Array.fill(1000)(freshCollection())
+    zipped = xs.map(x => (x, x))
+    if (size > 0) {
+      randomIndices = scala.Array.fill(1000)(scala.util.Random.nextInt(size))
+      randomIndices2 = scala.Array.fill(1000)(scala.util.Random.nextInt(size))
+      randomXss = scala.Array.fill(1000)(freshCollection().take(scala.util.Random.nextInt(size)))
+    }
   }
 
   @Benchmark
-//  @OperationsPerInvocation(size)
-  def cons(bh: Blackhole): Unit = {
+  def incl(bh: Blackhole): Unit = {
     var ys = HashSet.empty[Long]
     var i = 0L
     while (i < size) {
       ys = ys + i
-      i = i + 1
+      i += 1
```

Comment:
We should be consistent with style.  Now `i` and `ys` are updated with different styles.  Either is fine, as long as they're both that way.